rplugin/python3/denite/source/grep.py

```diff
@@ -228,8 +228,10 @@ def _init_patterns(self, context: UserContext,
         elif context['input']:
             patterns = [context['input']]
         else:
-            patterns = self.vim.call('denite#util#input',
-                                     'Pattern: ',
-                                     '',
-                                     '')
+            patterns = [
+                self.vim.call('denite#util#input',
+                              'Pattern: ',
+                              '',
+                              '')
+            ]
```

Comment:
The old implementation.
https://github.com/Shougo/denite.nvim/pull/722/commits/4af5416f58573f166db918a7f9dd4e8f7b4898eb#diff-65f6307045e802e7574524f64bf11496L228